mundane-control/src/main/scala/com/ambiata/mundane/control/RIO.scala

```diff
@@ -72,6 +74,17 @@ object RIO {
   def addFinalizer(f: Finalizer): RIO[Unit] =
     RIO(finalizers => { finalizers.add(f); ResultT.unit })
 
+  def unsafeFlushFinalizers: RIO[Unit] = RIO(x => {
+    ResultT.safe[IO, Unit](x.asScala.toList.foreach(_.run.unsafePerformIO match {
```

Comment:
Would we want to run all the finalizers even if one of them fails (and then throw an exception after)?